.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
check your package, JSON file to resolve this conflict  or re-install your linters and initialize a new packege.json file

Reply:
- [ ] Kindly update your review as per suggested by the previous reviewer.

Reply:
updated my review as per suggested by the previous reviewer 👍🏽